conda/cli/conda_argparse.py

```diff
@@ -285,11 +285,6 @@ def configure_parser_info(sub_parsers):
         action="store_true",
         help="List all known conda environments.",
     )
-    p.add_argument(
-        '-l', "--license",
-        action="store_true",
-        help="Display information about the local conda licenses list.",
```

Comment:
Rather than remove this altogether, we should just make it `help=SUPPRESS` for a while.  Just in case people have scripts that are using it.  Someday we might get around to writing deprecation warnings for all the `help=SUPPRESS` stuff.

Reply:
Ok